proto/options.proto

```diff
@@ -41,20 +41,17 @@ enum ShapeFormat {
 
 enum Costing {
   auto_ = 0;
-  // deprecated auto_shorter = 1;
```

Comment:
we dont need deprecated spots anymore because costing options has been converted to a map rather than a repeated field